cli/cmd/testdata/prom-config.yaml

```diff
@@ -35,3 +35,26 @@ prometheus:
     configMap: linkerd-prometheus-rules
   remoteWrite:
   - url: http://cortex-service.default:9009/api/prom/push
+  sidecarContainers:
+  - name: sidecar
+    lifecycle:
+      # n.b. only valid on k8s >=1.19
+      type: Sidecar
```

Comment:
We can leave this type as an example, but would you remove that comment? Sidecars won't make it to 1.19 (when they will is anybody's guess unfortunately). Otherwise we'll have to edit this out later on :wink:

Reply:
Sadness; removed.